registry/default/magicui/animated-subscribe-button.tsx

```diff
@@ -1,66 +1,88 @@
 "use client";
 
+import { HTMLMotionProps } from "motion/react";
 import { AnimatePresence, motion } from "motion/react";
 import React, { useState } from "react";
 
-interface AnimatedSubscribeButtonProps {
+interface AnimatedSubscribeButtonProps extends HTMLMotionProps<"button"> {
   buttonColor: string;
   buttonTextColor?: string;
```

Comment:
let's remove the buttoncolor and buttontext color props and prefer tailwind classes instead